pkg/gadgets/internal/socketenricher/tracer.go

```diff
@@ -58,14 +62,36 @@ func NewSocketEnricher() (*SocketEnricher, error) {
 }
 
 func (se *SocketEnricher) start() error {
+	specIter, err := loadSocketsiter()
+	if err != nil {
+		return fmt.Errorf("loading socketsiter asset: %w", err)
+	}
+
+	kallsyms.SpecUpdateAddresses(specIter, []string{"socket_file_ops"})
+
+	disableBPFIterators := false
+	if err := specIter.LoadAndAssign(&se.objsIter, nil); err != nil {
+		disableBPFIterators = true
+		log.Debugf("Socket enricher: skip loading iterators: %v", err)
```

Comment:
According to [this](https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels) SO answer:

"Warn - Anything that can potentially cause application oddities, but for which I am automatically recovering. (Such as switching from a primary to backup server, retrying an operation, missing secondary data, etc.)"

I think it perfectly matches this case, so this (and other similar ones in this commit) should be switched to warn.